README.md

```diff
@@ -9,6 +9,9 @@ A Netlify add-on is a way for Netlify users to extend their site functionality.
 - Setup `env` variables in Netlify's build context for users Automatically
 - ...
 
+> **Tip:** 
```

Comment:
By the arrangement of the lines, I'm guessing you were aiming to make this on a separate line from the paragraph that follows. I checked it in the preview, though, and they wrap together because there's no empty line between them.

<img width="340" alt="tip-alignment" src="https://user-images.githubusercontent.com/6111186/90690076-ef057100-e225-11ea-9678-18aecee40fbb.png">

I actually kinda like it that way, but I'm totally cool with the separated look, too, if you prefer. In that case, I think you can make that happen by adding and extra line with the `>` , OR adding a `<br>` tag at the end of the first line. (The former strategy will probably add an extra line between the "Tip" and the paragraph, but I'm not positive the latter strategy will work as expected.)

Also, I'm not sure if you know about how to check the rich-text previews of file diffs in GitHub. You can find it hidden in an icon button at the top of the file:

<img width="308" alt="preview-button" src="https://user-images.githubusercontent.com/6111186/90692126-6983c000-e229-11ea-9613-50296c702be2.png">

Reply:
Thanks for the tip on the Rich Text diff. You're right that I was anticipating breaking up the lines, but I also like it better this way since there's no Tip icon.